packages/web3/test/integration/external-providers/in3.test.ts

```diff
@@ -24,7 +24,7 @@ import Web3 from '../../../src/index';
 // Actually it could take long time to get something from `in3` because of its decentralized nature.
 // 	And because of that, this test only try to get the last block number. And this also simplifies the configuration needed.
 describe('compatibility with `in3` provider', () => {
-	it('should get last block number', async () => {
+	it.skip('should get last block number', async () => {
```

Comment:
👍  should also create an issue for this, if we haven't already

Reply:
we don't need. it is just limit problem. so this test will work from 1st of june